### PR TITLE
commit of the day

### PR DESCRIPTION
**Pull request by annica83**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,14 +8,17 @@
     </head>
     <body>
         <header>
-            <h1>Menyer</h1>
+            <h1>Meny </h1>
```
Comment (XolyExperience) [inline index.html]: Du ändrade från 'Menyer' till 'Meny' vänligt ändra tillbaka.

File: style.css [hunk 2]
```diff
@@ -11,6 +11,10 @@ footer {
     max-width: 700px;
 }
 
+h1 {
+    color: deeppink;
```
Comment (XolyExperience) [inline style.css]: h1 ska inte vara 'hotpink' i main css, vänligen ändra tillbaka.

**Review by XolyExperience — CHANGES_REQUESTED** [review]
ändra meny till menyer i main index,
ta bort h1 formateringen i main css.

File: menyer/hellskitchen/style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    font-size: large;
+}
+a:link{
+    text-decoration: none;
+    color: darksalmon;
+}
+a.hover{
+    text-decoration: underline;
+}
+
```
Comment (XolyExperience) [inline menyer/hellskitchen/style.css]: Do a little code cleanup and remove empty lines

**Review by XolyExperience — CHANGES_REQUESTED** [review]
Looks fine just do some cleanup and remove empty lines

**Review by XolyExperience — APPROVED** [review]
Looks great, awaiting Frozenbanana review and approval.

File: menyer/hellskitchen/style.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ul{
+    list-style: none;
+    padding: 0%;
+    margin: 20px;
+}
+li{
+    font-size: large;
+}
+a:link{
+    text-decoration: none;
```
Comment (frozenbanana) [inline menyer/hellskitchen/style.css]: Usually, you have one empty line between each selector.

File: menyer/hellskitchen/index.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    </header>
+    <main>
+        <div>
+            <h2>Förrätter</h2>
+            <article>
+                <img src="rakmacka.jpg" alt="logo">
+                <p>Räkmacka- med hemmagjort surdegs bröd <br> 95 kr</p>
+                </article>
+            <article>
+                <img src="vitlöksbröd.jpg" alt="logo">
```
Comment (frozenbanana) [inline menyer/hellskitchen/index.html]: Usually, non-english characters and spaces are avoided for file names.

**Review by frozenbanana — CHANGES_REQUESTED** [review]
Just a few small fixes otherwise you are good!

File: menyer/hellskitchen/style.css (hunk 4 above)
Comment (annica83) [reply menyer/hellskitchen/style.css]: okej

File: menyer/hellskitchen/index.html (hunk 5 above)
Comment (annica83) [reply menyer/hellskitchen/index.html]: okay :)

**Review by frozenbanana — APPROVED** [review]
Looks good!

**Pull request merged**